src/libs/tools/lighttable.c

```diff
@@ -256,33 +256,43 @@ void gui_init(dt_lib_module_t *self)
   gtk_box_pack_start(GTK_BOX(self->widget), d->layout_box, TRUE, TRUE, 0);
 
   d->layout_filemanager = dtgtk_togglebutton_new(dtgtk_cairo_paint_lt_mode_grid, CPF_STYLE_FLAT, NULL);
+  dt_action_define(&darktable.view_manager->proxy.lighttable.view->actions, NULL,
+                   "toggle filemanager layout", d->layout_filemanager, NULL);
```

Comment:
Don't we need N_( ) macro around all label here?

In current code there is N_() on some and nothing on some others. We should have consistent definition here and I think N_() is better as it ensure that if this text is only present here it will get translated.

Reply:
In this case, dt_action_define is only used to link a widget to an action that was set up by 
`  dt_accel_register_lib_as_view("lighttable", NC_("accel", "toggle filemanager layout"), 0, 0);
`
Since the action already exists, the translated label does not need to be generated. This is similar to all the dt_accel_connect calls, that also do not need N_ or NC_.

Since the translation is in the "accel" context, because it is one of the legacy dt_accel_register calls, using N_ just would force translators to supply an _additional_ (duplicate) translation that will never be used (or tested).

Obviously, I agree with you that this is all confusing and needs cleaning up, but I'd still rather do that as part of the large refactor that we discussed before and that I don't have the bandwidth for right now.

In the mean time I think the correct priority is to link all widgets into the new framework somehow, even if not in the cleanest way internally, because that is a user visible aspect and gaps in coverage would be annoying. Let me know you don't agree.

I've now also added the navigation and histogram toggles (but not the color toggles because they have been set up in a confusing way that needs to be disentangled at some point) in a similar fashion.

Reply:
> Since the action already exists,

Sure, but as soon one is renamed or slightly changed and not the other we will break the translation. So somehow I feel that we should have `N_()` around all string in `dt_action_define().` But ok, let's do that in a separate PR.